backend/src/main/java/com/shipping/backend/services/ShippingRequestSenderImpl.java

```diff
@@ -0,0 +1,36 @@
+package com.shipping.backend.services;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.amqp.rabbit.core.RabbitTemplate;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+@Service
+public class ShippingRequestSenderImpl implements  ShippingRequestSender{
+
+    private final static Logger log = LoggerFactory.getLogger(ShippingRequestSenderImpl.class);
+
+    @Value("${config-queue.name}")
+    private String QUEUE_NAME;
+
+    @Value("${config-queue.exchange}")
+    private String QUEUE_EXCHANGE;
+
+    @Value("${config-queue.routing-key}")
+    private String QUEUE_ROUTING_KEY;
```

Comment:
This comment applies to `QUEUE_NAME` and `QUEUE_EXCHANGE` as well
Are this constants? Are this just class variables?

Reply:
they suppose to be constants comming from the properties file, but now are totally useless becasue of the configuration that I have, so now they will be removed